src/Benchmarks/Data/EfDb.cs

```diff
@@ -49,9 +57,10 @@ public Task<World> LoadSingleQueryRow()
             for (var i = 0; i < count; i++)
             {
                 var id = _random.Next(1, 10001);
-                var result = await _dbContext.World.AsTracking().FirstAsync(w => w.Id == id);
+                var result = await _firstWorldQuery(_dbContext, id);
+
+                _dbContext.Attach(result).Property("RandomNumber").CurrentValue = _random.Next(1, 10001);
```

Comment:
@mikeharder I think this should work for now, can you try it?:

1. Change the compiled query declared in the class to be tracked explicitly:
``` C#
private static readonly Func<ApplicationDbContext, int, Task<World>> _firstWorldQuery 
    = EF.CompileAsyncQuery((ApplicationDbContext context, int id)  
        => context.World.AsTracking().First(w => w.Id == id)); 
```

2. Then change the code inside the loop to do:
 ``` C#
var id = _random.Next(1, 10001); 
var result = await _firstWorldQuery(_dbContext, id); 
_dbContext.Entry(result).Property("RandomNumber").CurrentValue = _random.Next(1, 10001); 
...
```